app/models/market.rb

```diff
@@ -0,0 +1,13 @@
+class Market < ApplicationRecord 
+	validates :name, :street, :city, :county, :state, :zip, :lat, :lon, presence: { message: "%{attribute} can't be blank." }
+
+	attr_reader :vendor_count
```

Comment:
Do we need an attr_reader here for the vendor_count since it's written as a method for the model? Perhaps I'm not seeing why it would need it just yet!